webpack.config.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-env node */
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack = require('webpack');
+const path = require('path');
+
+const staticFilesDir = path.resolve('contentcuration', 'contentcuration', 'static');
+const staticJsDir = path.resolve(staticFilesDir, 'js');
+const staticLessDir = path.resolve(staticFilesDir, 'less');
+
+const bundleEntryDir = path.resolve(staticJsDir, 'bundle_modules');
+const bundleOutputDir = path.resolve(staticJsDir,'bundles');
+
+const jsLoaders = [
+  {
+    loader: 'babel-loader',
+    options: {
+      // might be able to limit browsers for smaller bundles
+      presets: ['env'],
+    },
+  }
+];
+
+// NOTE: Lots of things are handled by webpack4. NODE_ENV, uglify, source-maps
+// see: https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a
+
+module.exports = {
+  context: bundleEntryDir,
+  entry: {
+    base: './base.js',
+    channel_edit: './channel_edit.js',
+  },
+  output: {
+    filename: '[name].js',
```

Comment:
Should change this to `[name]-[hash].js'` for cache busting builds.

Reply:
Any suggestions for the best way to include this in the django template's `<script>`?